pkgs/stdenv/generic/setup.sh

```diff
@@ -994,14 +1042,11 @@ substituteInPlace() {
 }
 
 _allFlags() {
-    # export some local variables for the awk below
-    # so some substitutions such as name don't have to be in the env attrset
-    # when __structuredAttrs is enabled
+    # Export some local variables for the `awk` below so some substitutions (such as name)
+    # don't have to be in the env attrset when `__structuredAttrs` is enabled.
     export system pname name version
     while IFS='' read -r varName; do
-        if (( "${NIX_DEBUG:-0}" >= 1 )); then
-            printf "@%s@ -> %q\n" "${varName}" "${!varName}" >&2
-        fi
+        nixWarnLog "@${varName}@" "->" "${!varName}"
```

Comment:
```suggestion
        nixWarnLog "@${varName}@ -> ${!varName}"
```

Reply:
Not sure if this should be a `warn` level. This seems to be `talkative` as well.
Though this would increase the level of NIX_DEBUG required for this message.

Reply:
Yeah, I kept it due to the previous code. None of the logging under NIX_DEBUG looks like it's tuned for levels, and this one in particular isn't.